content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md

```diff
@@ -0,0 +1,304 @@
+        <td><b>Operations</b></td>
+    </tr>
+</table>
+<hr />
+
+|                    | **Create** | **Delete** | **Get** | **List** | **Update** | **Import** | **Publish** | **Backup** | **Restore** |
+| ------------------ | ---------- | ---------- | ------- | -------- | ---------- | ---------- | ----------- | ---------- | ----------- |
+| **clusterProfile** |            |            | √       | √        |            |            |             |            |             |
+| **macro**          |            |            | √       | √        |            |            |             |            |             |
+| **packRegistry**   |            |            | √       | √        |            |            |             |            |             |
+
+<br />
+
+</Tabs.TabPane>
+</Tabs>
+
+<br />
+
+
+
+# Palette Custom Resource Roles
+
+<br / >
+
+The following is a list of platform permissions and operations supported by Palette. Use these permissions to [create custom role](/user-management/new-user#createcustomrole) to control the cluster access. For every **Resource Keys** available **operations** can be added as per your requirements.
+
+<br / >
+
+## List of Custom Permissions
+
+<br / >
+<table>
+    <tr>
+        <td width="400"><b>resourceKeys</b></td>
+        <td><b>Operations</b></td>
+    </tr>
+</table>
+<hr />
+
+|                    | **Create** | **Delete** | **Get** | **List** | **Update** | **Import** | **Publish** | **Backup** | **Restore** |
+| ------------------ | ---------- | ---------- | ------- | -------- | ---------- | ---------- | ----------- | ---------- | ----------- |
+| **cloudaccount**   |            |            | √       | √        |            |            |             |            |             |
+| **cloudconfig**    |            | √          | √       | √        | √          |            |             |            |             |
+| **cluster**        |            | √          | √       | √        | √          |            |             |            |             |
+| **clusterProfile** |            | √          | √       | √        | √          |            | √           |            |             |
+| **dnsMapping**     |            |            | √       | √        |            |            |             |            |             |
+| **location**       |            |            | √       | √        |            |            |             |            |             |
+| **machine**        |            |            | √       | √        |            |            |             |            |             |
+| **macro**          |            |            | √       | √        |            |            |             |            |             |
+| **packRegistry**   |            |            | √       | √        |            |            |             |            |             |
+
+
+## Resources
+
+[Resource Scope Matrix](http://localhost:9000/user-management/palette-rbac#resourcescopematrix)
```

Comment:
```suggestion
[Resource Scope Matrix](/user-management/palette-rbac#resourcescopematrix)
```
resma, I shouldn't have to fix simple issues like this. Slow down and review your changes.